make/verify-dependencies.mk

```
@@ -0,0 +1,16 @@
+.PHONY: verify-dependencies
+## Runs commands to verify after the updated dependecies of toolchain-common/API(go mod replace), if the repo needs any changes to be made
+verify-dependencies: tidy vet compile test lint-go-code
+
+.PHONY: tidy
+tidy: 
+	go mod tidy
+
+.PHONY: vet
+vet:
+	go vet ./...
+
+.PHONY: compile
+compile:
```

`compile` can be a bit confusing
```suggestion
.PHONY: go-test-skip-all
go-test-skip-all:
```

---

done